src/ReCaptcha.php

```diff
@@ -241,6 +241,10 @@ public function getParams()
      */
     public function getParam($key)
     {
+        if (! isset($this->params[$key])) {
```

Comment:
This condition is added to fix tests, previously there was typo in test methods, see: https://github.com/zendframework/ZendService_ReCaptcha/pull/13/files#diff-2e1212952b04d9b7cc6e538238eb2279L67